src/composables/useContext.ts

```diff
@@ -0,0 +1,146 @@
+import {computed, reactive, toRefs} from '@vue/composition-api';
+import {contextSchema} from '@/lib/ContextValidator';
+import {Context, Shop, ShopContext} from '@/types/context';
+
+const defaultContext = (): Partial<Context> => ({
+  // psIs17: true,
+  psAccountsInstallLink: null,
+  psAccountsEnableLink: null,
+  psAccountsUpdateLink: null,
+  psAccountsIsInstalled: false,
+  psAccountsIsEnabled: false,
+  psAccountsIsUptodate: false,
+  onboardingLink: null,
+  user: {
+    email: null,
+    emailIsValidated: false,
+    isSuperAdmin: false,
+  },
+  currentShop: null,
+  shops: [],
+  superAdminEmail: null,
+  ssoResendVerificationEmail: null,
+  manageAccountLink: null,
+  errors: undefined,
+});
```

Comment:
I think we need to remove the "defaultContext" that I added some time ago, what do u think ?